cmd/crossplane/core/core.go

```diff
@@ -78,6 +78,11 @@ type startCommand struct {
 
 	EnableCompositionRevisions bool `group:"Alpha Features:" help:"Enable support for CompositionRevisions."`
 	EnableExternalSecretStores bool `group:"Alpha Features:" help:"Enable support for ExternalSecretStores."`
+	// NOTE(hasheddan): this feature is unlikely to graduate from alpha status
+	// and should be removed when a runtime interface is introduced upstream.
+	// See https://github.com/crossplane/crossplane/issues/2671 for more
+	// information.
+	EnableProviderIdentity bool `group:"Alpha Features" help:"Enable support for Provider identity."`
```

Comment:
```suggestion
	EnableProviderIdentity bool `group:"Alpha Features:" help:"Enable support for Provider identity."`
```
Based on formatting of the above flags 👍.

Reply:
ah nice catch!